tools/gcc.py

```diff
@@ -162,7 +162,7 @@ def checkAndGetResult(pattern, string):
             result += '#define HAVE_SIGVAL 1\n'
 
         if version:
-            result += '#define GCC_VERSION_STR "%s"\n' % version
+            result += '#define GCC_VERSION "%s"\n' % version
```

Comment:
Why modify here?

Reply:
When I used the toolchain from here: https://github.com/ilg-archived/riscv-none-gcc/releases ,
I failed to build it without this change, so maybe there is something wrong with my config?

Reply:
this is the error I met without this patch, 
`In file included from /home/tekkamanninja/development/RISC-V/K210/SDK/rt-thread/include/rtthread.h:24,
                 from /home/tekkamanninja/development/RISC-V/K210/SDK/rt-thread/components/drivers/include/drivers/pin.h:15,
                 from /home/tekkamanninja/development/RISC-V/K210/SDK/rt-thread/components/drivers/misc/pin.c:11:
/home/tekkamanninja/development/RISC-V/K210/SDK/rt-thread/include/rtdef.h:471:9: error: unknown type name 'siginfo_t'
 typedef siginfo_t rt_siginfo_t;
         ^~~~~~~~~
scons: *** [build/kernel/components/drivers/misc/pin.o] Error 1`

Reply:
There won't be this problem if we apply this patch, so please let me know if I misunderstand this or nor.

Reply:
The GCC_VERSION / GCC_VERSION_STR should not cause this issue.  If you change the compiler, please remove cconfig.h file or `scons -c`, then build rt-thread again.

Reply:
Hi BernardXiong, 
I have just tried it , it doesn't work, the problem still there.  :-(

Reply:
pushd bsp/k210/
scons  --menuconfig
source ~/.env/env.sh
pkgs --update
scons
popd

anything wrong here ??? :-)